### ISSUE TITLE
Error on iPhone 14 Pro

### ISSUE THREAD

**Issue by lucabordignonb** [yandevelop/Bea#27]
<img width="1130" alt="Screenshot 2023-11-08 alle 15 39 49" src="https://github.com/yandevelop/Bea/assets/119453703/00610a4b-b03e-4d3d-bc0c-6e323d75f082">

Why? It should be compatible with all devices...

**Comment by yandevelop** [comment]
I forgot to add devices to supported list lol. The issue is on my side i'll try to fix it

**Comment by lucabordignonb** [comment]
Thanks for the reply. Does the sideload version work if I download now from
https://github.com/yandevelop/Bea ?

Thanks again!

Luca

Il giorno gio 9 nov 2023 alle ore 14:52 yan ***@***.***> ha
scritto:

**Comment by yandevelop** [comment]
No, I usually have a script that adds the devices "missing" in the Info.plist, but I forgot to execute it before uploading the latest ipa to the Releases. I'll do it as soon as I get to it

**Comment by lucabordignonb** [comment]
I did it and solved the problem. Unfortunately when I do the configuration I get this error and I can't proceed..
![IMG_0194](https://github.com/yandevelop/Bea/assets/119453703/e7929a9f-1183-4231-881d-f5c07151c296)

**Comment by yandevelop** [comment]
Ok, never seen that error and this is again on BeReal's side

**Comment by lucabordignonb** [comment]
I'll try to wait and see if the situation changes.. I don't know what I can do 

thanks!

**Comment by yandevelop** [comment]
no worries. look on r/BeReal_App if someone posted something there. I saw similar posts that had the same problem with SMS Authorization. Alternatively you can reach out to BeReal support.

**Comment by takunnma5286** [comment]
![image](https://github.com/yandevelop/Bea/assets/109655207/46a6d40a-7f18-455e-b11e-6f53045e2cd2)
I had a same error on iPhone SE
